scripts/lib/queue_workers.py

```diff
@@ -6,7 +6,7 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 sys.path.append(BASE_DIR)
-import scripts.lib.setup_path_on_import
+import scripts.lib.setup_path_on_import  # WARN this actually runs a script
```

Comment:
I think this surprise should be discouraged. A function should be imported, then run, instead of having the import statement having a side effect of executing things.

Reply:
The name does suggest what it does.  But it's possible we should make it a 2-line thing, where you import it and then call some code from it.